### DISCUSSION TITLE
DA Scroll Speed rounding to nearest tenth everywhere except Customisation menu

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#17045] [Q&A] [answered]
Version 2022.226.0
When Scroll x1.35 is enabled, it displays as Scroll x1.4 on loading the beatmap, the results screen, and the leaderboard. This looks to be a recent regression as I can't remember this happening a month ago.
https://www.twitch.tv/videos/1412785686?t=0h38m35s

**Comment by peppy** [comment, marked as answer]
If possible could you please post screenshots of where you are seeing this? I can't see it via twitch because of ads.

**Reply by peppy** [reply]
Should be resolved via #17048.

**Reply by zachmanthethird** [reply]
Super sorry; I always forget about Twitch ads. Will do in the future :+1:
